packages/atlas/src/components/_overlays/ImageCropModal/ImageCropModal.stories.tsx

```diff
@@ -62,7 +62,7 @@ const RegularTemplate: Story<ImageCropModalProps> = () => {
 
   return (
     <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'start', gap: '24px' }}>
-      <Avatar assetUrl={avatarImageUrl} editable onEditClick={() => avatarDialogRef.current?.open()} size="cover" />
+      <Avatar assetUrl={avatarImageUrl} editable onClick={() => avatarDialogRef.current?.open()} size="cover" />
```

Comment:
```suggestion
      <Avatar assetUrl={avatarImageUrl} editable onClick={avatarDialogRef.current?.open} size="cover" />
```

Reply:
This would produce TS errors since `onClick` will pass `MouseEvent` as an argument and that's not what `open` expects